scope_test.go

```diff
@@ -7,8 +7,11 @@ import (
 	"time"
 )
 
+//nolint: gochecknoglobals
```

Comment:
This lint rule is not useful. While minimizing global state is relevant, sprinkling code with lint comments is questionable.

`//nolint: gochecknoglobals` is 50% of the `//nolint:` comments in the code base.